app/controllers/searches_controller.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class SearchesController < ApplicationController
+  layout :set_layout, only: [:index]
```

Comment:
For now, the redesigned layout will be limited to the `#index` action. The other actions will be addressed in future PRs, and eventually the `only: [:index]` will be removed.

Reply:
Not a super strong opinion, but I tend to prefer to add comments for temporary code.

Reply:
This is actually going to be removed very soon, possibly in the next redesign PR, so I think we're fine without a comment in this case.